backend/src/project/dtos/update-status.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class UpdateStatusDTO {
+  @IsNotEmpty()
+  @IsString()
+  @ApiProperty({ required: true })
+  label?: string;
```

Comment:
```suggestion
  label: string;
```
znak zapytania = nie required a to jedyna rzecz przy update więc fajnie by było jej wymagać :D